docs/build.gradle

```diff
@@ -116,6 +116,15 @@ tasks.named("yamlRestTest").configure {
   }
 }
 
+// TODO: Remove the following when RCS feature is released
+// The get-builtin-privileges doc test does not include the new cluster privilege for RCS
+// So we disable the test if the build is a snapshot where unreleased feature is enabled by default
+tasks.named("yamlRestTest").configure {
+  if (BuildParams.isSnapshotBuild()) {
+    systemProperty 'tests.rest.blacklist', 'reference/rest-api/usage/*'
+  }
+}
```

Comment:
Since RCS is still behind the feature flag, we don't want to change the doc test to show the new usage section. Therefore we just have to skip this particular test on snapshot builds (still runs on release builds).

Reply:
what would be the cons around updating the docs to show the new section? I was under the impression that it's useful regardless of whether RCS is released or not?

Reply:
The new section is behind a feature flag. I think we can argue that some of these information does not have to behind the feature flag since it is not new. But the effort to conditionally include parts of it does not seem to justify. So it is hidden entirely for the time being.